bin/apisix

```diff
@@ -706,28 +706,33 @@ local function init_etcd(show_output)
     end
 
     local etcd_conf = yaml_conf.etcd
-    local uri = etcd_conf.host .. "/v2/keys" .. (etcd_conf.prefix or "")
 
     local timeout = etcd_conf.timeout or 3
+    local uri
 
-    for _, dir_name in ipairs({"/routes", "/upstreams", "/services",
-                               "/plugins", "/consumers", "/node_status",
-                               "/ssl", "/global_rules", "/stream_routes",
-                               "/proto"}) do
-        local cmd = "curl " .. uri .. dir_name
+    for _, host in ipairs(yaml_conf.etcd.host) do
+        uri = host .. "/v2/keys" .. (etcd_conf.prefix or "")
+
+        for _, dir_name in ipairs({"/routes", "/upstreams", "/services",
```

Comment:
The current process is wrong.

That's fake code, please take a look.

```lua
-- fake code
for i, host in ipairs(yaml_conf.etcd.host) do
    local is_success = true
    for _, dir_name in ipairs({"/routes", "/upstreams", ...}) do
        local ok = ...
        if not ok then
            is_success = false
            break
        end
    end

    if is_success then
        break
    end
end
```

Reply:
@membphis fixed this